src/containers/NFT/NFTTabs/test/NFTTabs.test.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { I18nextProvider } from 'react-i18next'
+import { MemoryRouter as Router, Route } from 'react-router-dom'
+import { QueryClient, QueryClientProvider } from 'react-query'
+import { NFTTabs } from '../NFTTabs'
+import i18n from '../../../../i18nTestConfig'
+
+describe('NFT Transactions tab container', () => {
+  const queryClient = new QueryClient({
```

Comment:
The defining of the client should be pulled out into its own file since it is the same as in App.js and in every test that has react-query in it.